plugins/_collections.js

```diff
@@ -1679,6 +1680,69 @@ exports.elems = {
       contentScriptType: 'application/ecmascript',
       contentStyleType: 'text/css',
     },
+    deprecated: [
```

Comment:
This list is hard-coded for the `<svg>` node only, but most of these attributes aren't for the `<svg>` node at all, and were already being removed by SVGO as unknown attributes.

For example, `accent-height` is used by the `<font-face>` node, not `<svg>`.

The deprecated attributes should be split into the separate nodes they were usable in. i.e. `accent-height` in `font-face` and `viewTarget` in `view`, etc.

Your current test only passes because it happens to test on an `<svg>` node, but a test on a `<view>` node fails:

```svg
Removes deprecated viewTarget attribute from view node.

===

<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
  <view id="one" viewBox="0 0 100 100" viewTarget=""/>
</svg>

@@@

<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
    <view id="one" viewBox="0 0 100 100"/>
</svg>
```